$OpenBSD: patch-src_libpiano_piano_c,v 1.3 2011/12/08 18:27:19 dcoppa Exp $

Fix memset (upstream git commit 2da38556d614b9bc969f65b3683c2ce6eb99e6bc)

--- src/libpiano/piano.c.orig	Thu Dec  8 19:20:36 2011
+++ src/libpiano/piano.c	Thu Dec  8 19:21:32 2011
@@ -92,7 +92,7 @@ void PianoDestroyStation (PianoStation_t *station) {
 	free (station->name);
 	free (station->id);
 	free (station->seedId);
-	memset (station, 0, sizeof (station));
+	memset (station, 0, sizeof (*station));
 }
 
 /*	free complete station list
